css/styles.css

```diff
@@ -0,0 +1,153 @@
+/* General Styles */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Inter, sans-serif;
+  background: #e5e5e5;
+}
+
+/* Media queries */
+@media (min-width: 375px) {
+  body {
+    width: 375px;
+  }
+}
+
+@media (min-width: 768px) {
+  body {
+    width: 768px;
+  }
+}
```

Comment:
- [ ] You did a brilliant job 👏  However, your design is void of the background images present on the figma template. Kindly add it up to compliment the good work you've done so far, and have a better user interface 👍 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/219823274-4f2aec2d-02e4-40c6-9dba-8b5ffe29e19a.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/219823479-af05e600-4bd8-4357-ba32-c3d0f503dbc7.png)